app/views/hosts/_provisioning.html.erb

```diff
@@ -4,7 +4,7 @@
   <h4><%= _('Templates resolved for this operating system') %></h4>
   <% templates.sort{|t,x| t.template_kind <=> x.template_kind}.each do |tmplt| %>
       <div class="form-group">
-        <div class="control-label col-md-8"> <%= _("%s Template") % tmplt.template_kind %>
+        <div class="control-label col-md-8"> <%= _(tmplt.template_kind) %>
```

Comment:
Do we need to translate it?..

Reply:
Oh good point, previously it was untranslated.

Reply:
Oh, true, we should probably not want to translate this.